### PR TITLE
Feature/multiroom

### PR DESCRIPTION
**Pull request by am0s**
Adds support for multi-room casting as well as improved details about Audio cast devices.
The main thing to support multi-room is to use the port value reported by the cast device.

### CODE REVIEW

File: pychromecast/__init__.py [hunk 1]
```diff
@@ -161,31 +178,72 @@ class Chromecast(object):
...
+                                  dev_status.manufacturer),
+                    api_version=(device.api_version or
+                                 dev_status.api_version),
+                    uuid=(device.uuid or
+                          dev_status.uuid),
+                    cast_type=(device.cast_type or
+                               dev_status.cast_type),
+                )
+            else:
+                self.device = DeviceStatus(
```
Comment (balloob) [inline pychromecast/__init__.py]: This seems like a one on one copy, can't we just omit it ?

**Comment by balloob** [comment]
Looks very solid, nice job! :dolphin: 

Just one minor comment but this looks good to go.

File: pychromecast/__init__.py (hunk 1 above)
Comment (am0s) [inline pychromecast/__init__.py]: It merges the two objects as we don't get all information from the dial event.
The remaining parts are filled in from the device description.
The dialog information is the most correct but it does not have manufacturer I believe it was.

But I'll see if I can rewrite this code to be a bit more elegant + a comment about this.

Comment (balloob) [inline pychromecast/__init__.py]: I see the merging in the above statement but isn't the one below only using data from `device` ?

Comment (am0s) [inline pychromecast/__init__.py]: Ah, I did that to get a copy but since these are tuples there's no point in that. I'll fix.

**Comment by am0s** [comment]
Ok, updated the code for DeviceStatus, and also added a version variable which I noticed was missing

**Comment by balloob** [comment]
Ok it looks all good. You can merge when you're ready. Want to push out a new version too?

**Comment by am0s** [comment]
Sure, would this be a 0.7 release?

**Comment by balloob** [comment]
I would imagine so, pretty significant feature bump

**Pull request merged**